k8s/src/main/scala/io/buoyant/k8s/Stabilize.scala

```diff
@@ -0,0 +1,60 @@
+package io.buoyant.k8s
+
+import com.twitter.util.{Activity, Updatable, Var}
+
+trait Stabilize {
```

Comment:
Inheritance is a powerful tool that we don't actually need here.  `stabilize` is just a static function.  This is the first step down a dark path that eventually leads to the cake pattern.